src/components/results/common/global-filter/overflow-tooltip.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { PropsWithChildren, useEffect, useRef, useState } from 'react';
+import { Box } from '@mui/material';
+import Tooltip from '@mui/material/Tooltip';
+
+export interface OverflowTooltipProps extends PropsWithChildren {
+    label: string;
+}
+
+/**
+ * if the label is too long (reaches overflow) a tooltip is added to display the full text
+ */
+function OverflowTooltip({ label }: Readonly<OverflowTooltipProps>) {
+    const [displayTooltip, setDisplayTooltip] = useState(false);
+    const textElementRef = useRef<HTMLInputElement | null>(null);
+
+    const compareSize = () => {
+        if (textElementRef.current) {
+            const compare = textElementRef.current.scrollWidth > textElementRef.current.clientWidth;
+            setDisplayTooltip(compare);
+        }
+    };
+
+    useEffect(() => {
```

Comment:
Replace 2 useEffects by : 
```suggestion
    useEffect(() => {
        compareSize();
        window.addEventListener('resize', compareSize);
        return () => {
            window.removeEventListener('resize', compareSize);
        };
    }, []);
```

Reply:
Yep, but I removed this class. Thang signaled me that OverflowableText in commons-ui does the same work so I used it.